TradePlay/src/components/modals/add-to-playlist-small-modal.jsx

```diff
@@ -0,0 +1,25 @@
+import { usePlaylist } from "../../context/playlist-context";
+
+const AddToPlaylistSmallModal = ({ videoId, switchModal }) => {
+  const { playlistState, playlistDispatch } = usePlaylist();
+  const clickHandler = ({ name }) => {
```

Comment:
Try naming the function with a meaningful name.

Reply:
Yes I will keep that on mind from now on